generator/source/release/bumper/notes.md

```diff
@@ -12,8 +12,8 @@ release-name: bumper
 
 <a href="top"></a>
 
-Hyperledger Sawtooth 1.1 (Bumper) is now available. See the [latest
-documentation](https://sawtooth.hyperledger.org/docs/core/releases/latest/) to
+Hyperledger Sawtooth 1.1 (Bumper) is now available. See the
+[documentation](https://sawtooth.hyperledger.org/docs/core/releases/1.1.5/) to
```

Comment:
Shouldn't this be releases/1.1/? (w/link on the filesystem)

Reply:
See my above comment to @danintel